modules/google-analytics/wp-google-analytics.php

```diff
@@ -50,7 +50,18 @@ class Jetpack_Google_Analytics {
 	 * @return void
 	 */
 	private function __construct() {
-		if ( Jetpack_Google_Analytics_Options::enhanced_ecommerce_tracking_is_enabled() ) {
+		/**
+		 * Allow for universal analytics (analytics.js) to be used outside of ecommerce contexts
+		 *
+		 * @since 6.5.0
+		 *
+		 * @param bool Whether universal analytics (analytics.js) should be used (true) or legacy (ga.js) should be used (false)
+		 */
+		$use_universal_analytics = apply_filters( 'jetpack_wga_use_universal_analytics',
+			Jetpack_Google_Analytics_Options::enhanced_ecommerce_tracking_is_enabled()
```

Comment:
Seems like this filter is defaulting to `true` if enhanced ecommerce is enabled. so the provided filter in the testing instructions would not affect the truthiness we're getting after the step: `Enable Google Analytics Enhanced eCommerce Analytics in Jetpack Settings`.

Please @allendav  clarify. @brbrr comments about not seeing `ga.js` being logged come from this same fact.